preview_generator/preview/builder/video__ffmpeg.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+
+import typing
+import ffmpeg
+import json
+
+from preview_generator import utils
+from preview_generator.preview.generic_preview import (
+    PreviewBuilder,
+)
+
+from preview_generator.exception import (
+    PreviewGeneratorException,
+)
+
+
+class NoVideoStream(PreviewGeneratorException):
+    pass
+
+
+class VideoPreviewBuilderFFMPEG(PreviewBuilder):
+    page_nb = 10
+
+    @classmethod
+    def get_label(cls) -> str:
+        return "Video files - based on ffmpeg"
+
+    @classmethod
+    def get_supported_mimetypes(cls) -> typing.List[str]:
+        return [
+            "application/x-videolan",
+            "video/3gpp",
+            "video/annodex",
+            "video/dl",
+            "video/dv",
+            "video/fli",
+            "video/gl",
+            "video/mpeg",
+            "video/MP2T",
+            "video/mp4",
+            "video/quicktime",
+            "video/mp4v-es",
+            "video/ogg",
+            "video/parityfec",
+            "video/pointer",
+            "video/webm",
+            "video/vnd.fvt",
+            "video/vnd.motorola.video",
+            "video/vnd.motorola.videop",
+            "video/vnd.mpegurl",
+            "video/vnd.mts",
+            "video/vnd.nokia.interleaved-multimedia",
+            "video/vnd.vivo",
+            "video/x-flv",
+            "video/x-la-asf",
+            "video/x-mng",
+            "video/x-ms-asf",
+            "video/x-ms-wm",
+            "video/x-ms-wmv",
+            "video/x-ms-wmx",
+            "video/x-ms-wvx",
+            "video/x-msvideo",
+            "video/x-sgi-movie",
+            "video/x-matroska",
+        ]
+
+    def get_dims_from_ffmpeg_probe(
+        self, probe_result: dict
```

Comment:
Is it possible to be more specific about typing of dict here ?
```suggestion
        self, probe_result: typing.Dict[str, typing.Any]
```